lib/reports/oclc_registration.rb

```diff
@@ -8,30 +8,30 @@ module Reports
   # Runs a report for an organization, outputting all of their
   # shared print commitments in an OCLC Registration-like format.
   class OCLCRegistration
-    attr_reader :output_file
     def initialize(organization)
       @organization = organization
       @outd = Settings.oclc_registration_report_path
       @collection_id_map_path = Settings.oclc_collection_id_map_path
       validate!
       @collection_memo = {}
+      @finder = SharedPrint::Finder.new(organization: [@organization])
```

Comment:
This refactoring conceptually makes sense, but I wonder if it might make sense to extract finder to a method that can be overridden, rather than setting it in the parent class & then re-setting it in the child? e.g.

```ruby
class OCLCRegistration
  def finder
    SharedPrint::Finder.new(organization: [@organization])
  end
end

class Phase3OCLCRegistration < Reports::OCLCRegistration
  def finder
    SharedPrint::Finder.new(organization: [@organization], phase: [3])
  end
end
```

`finder`/`@finder` is only referred to once (in `run`), so this doesn't even really need to be an instance variable.

Reply:
Yes, makes more sense. Plus that means I can get rid of the constructor in the phase3report. 
It is done.